### PR TITLE
WIP: Feature/add account page

### PR DESCRIPTION
**Pull request by kinda-neat**

### CODE REVIEW

File: src/utils/format/getDecimal.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  const minPrecision =
+    baseDecimals - precision > positiveValue.length - 1
+      ? baseDecimals - positiveValue.length + 1
+      : precision;
+  const decimalsZerosLength = baseDecimals < minPrecision ? baseDecimals : minPrecision;
+
+  const fractional = `${`${'0'.repeat(padding)}${positiveValue}`.substr(
+    mid < 0 ? 0 : mid,
+  )}000`.substr(0, decimalsZerosLength);
```
Comment (sk1e) [inline src/utils/format/getDecimal.ts:20]: hard to read, should be split somehow

**Review by sk1e — CHANGES_REQUESTED** [review]

File: src/app/pages/Account/AccountPage.tsx [hunk 2]
```diff
@@ -1,33 +1,84 @@
...
-            <MyLoans account={account} />
-          </Grid>
-        </Grid>
-      )}
-    </WithAccount>
+    <Grid className={classes.root}>
+      <TabContext value={selectedPage}>
+        <TabsList value={selectedPage} className={classes.tabs} onChange={handleTabChange}>
+          <Tab
+            label="My Summary"
```
Comment (Everrest1) [inline src/app/pages/Account/AccountPage.tsx]: Should be used by translate util

**Review by sk1e — DISMISSED** [review]

File: src/app/pages/Account/AccountPage.tsx [hunk 3]
```diff
@@ -1,33 +1,84 @@
...
+    },
+    tab: {
+      minWidth: 112,
+    },
+  }),
+  { name: 'AccountPage' },
+);
+
+function makeUnimplementedComponent(componentLabel: string) {
+  return () => <div style={{ fontSize: 45 }}>{`${componentLabel} not implemented`}</div>;
```
Comment (Everrest1) [inline src/app/pages/Account/AccountPage.tsx:87]: Should be used by translate util too

File: src/components/FormattedAmount/FormattedAmount.tsx [hunk 4]
```diff
@@ -1,28 +1,88 @@
...
-  children?: FunctionComponent<{ formattedBalance: string; notRoundedBalance: string }>;
 }
 
 function FormattedAmount(props: IProps) {
-  const { sum, precision, children, className } = props;
+  const { sum, hideSymbol, precision = 2, className } = props;
   const formattedBalance = sum.toFormattedString(precision);
-  const notRoundedBalance = sum.toFormattedString(sum.currency.decimals);
+  const notRoundedBalance = sum.toFormattedString(
+    sum instanceof PercentAmount ? 5 : sum.currency.decimals,
```
Comment (Everrest1) [inline src/components/FormattedAmount/FormattedAmount.tsx]: Not sure, but 5 looks like magic number

**Review by in19farkt — APPROVED** [review]

**Pull request merged**